capsule-sync-check.sh

```diff
@@ -32,6 +39,13 @@ then
     sleep 300
     count_capsule_syncs_in_progress
     warn "${_CAPSULE_SYNC_COUNT} sync jobs to capsules still running"
+    ((WAIT+=300))
```

Comment:
I wonder if it would make sense to make this `300` configurable too? (Irrelevant to the main purpose of the PR)

Reply:
yeah, I thought about that but in the end decided against it.